src/server/helpers/source-api-request.ts

```diff
@@ -178,6 +194,25 @@ export async function requestData<T>(
 
     const request: AxiosPromise<T> = axiosRequest.request(requestConfig);
     const response: AxiosResponse<T> = await request;
+
+    // if we have a next link
+    if (
+      response.headers?.link?.indexOf('rel="next"') > 0 &&
```

Comment:
Als er 1.000.000 resultaten zijn gaan we die automatisch ophalen, kun je hier iets verzinnen dat we dat voorkomen?

Reply:
En deze @janthijs  ?